tests/test_scrapy_redis.py

```diff
@@ -193,7 +193,7 @@ def test_scheduler(self):
 
     def test_scheduler_persistent(self):
         # TODO: Improve this test to avoid the need to check for log messages.
-        self.spider.log = mock.Mock(spec=self.spider.log)
+        self.spider.log = mock.Mock(spec=self.spider.logger)
```

Comment:
I think is better to keep log function here, something wired to use `log = ... logger`

Reply:
@Germey 
I think you're right, we should keep log function here for the time being.

Reply:
I've tried to use `log` and `logger`, they both work. 
But there's a little different, `log` utility need to import `logging` to show other types of message, or it only shows `DEBUG` message. One the other hand, `logger` utility can show all kind of message without import a third party library.

Reply:
But the usage of line 196 is wrong, should be `spider.logger.<log level>`.